README.md

```diff
@@ -0,0 +1,55 @@
+# stringify-package [![npm version](https://img.shields.io/npm/v/stringify-package.svg)](https://npm.im/stringify-package) [![license](https://img.shields.io/npm/l/stringify-package.svg)](https://npm.im/stringify-package) [![Travis](https://img.shields.io/travis/npm/stringify-package/latest.svg)](https://travis-ci.org/npm/stringify-package) [![AppVeyor](https://img.shields.io/appveyor/ci/npm/stringify-package/latest.svg)](https://ci.appveyor.com/project/npm/stringify-package) [![Coverage Status](https://coveralls.io/repos/github/npm/stringify-package/badge.svg?branch=latest)](https://coveralls.io/github/npm/stringify-package?branch=latest)
+
```

Comment:
Do you use a tool to generate these badges or do you copy/paste from existing projects?

Reply:
They were copied from an existing npm repo and renamed to point to this one. I believe most of them will be broken until this is merged and tagged and released because the data they're referencing doesn't exist yet.